package.json

```diff
@@ -27,5 +27,8 @@
     "packages": [
       "packages/*"
     ]
+  },
+  "dependencies": {
```

Comment:
Hey I think this dependency must be for dapp only ?

Reply:
Also are you using npm ? I've setup to use yarn. npm won't work well with yarn workspaces, could you remove `package-lock.json` ?

Reply:
Am not sure but trying to resolve them.